app/controllers/families_controller.rb

```diff
@@ -0,0 +1,38 @@
+class FamiliesController < ApplicationController
+  before_action :require_login
+
+  def index
+    @family = Member.where(family_id: session[:user_id])
```

Comment:
Calling your variable family is a very strong indicator to the reader that it represents an instance of family. This is actually a collection of members. You should consider a different name here.